src/platform/plugins/shared/discover/public/application/main/components/tabs_view/use_preview_data.ts

```diff
@@ -172,7 +172,7 @@ const getPreviewDataObservable = (
           }
 
           return {
-            status: getPreviewStatus(fetchStatus),
+            status: tabState.forceFetchOnSelect ? TabStatus.DEFAULT : getPreviewStatus(fetchStatus),
```

Comment:
It makes sense to use the default preview status when `forceFetchOnSelect === true` since whatever results the tab has are no longer valid.